lib/trino-parquet/src/main/java/io/trino/parquet/writer/ParquetWriters.java

```diff
@@ -103,7 +103,7 @@ static PrimitiveValueWriter getValueWriter(ValuesWriter valuesWriter, Type type,
             return new IntegerValueWriter(valuesWriter, type, parquetType);
         }
```

Comment:
We also need to ensure that `parquetTimeZone` is hard-coded to UTC when parquet writer is used through hudi/iceberg/delta, just like we're doing for the reader.

Reply:
Good point.

https://github.com/trinodb/trino/actions/runs/10014094980/job/27683136191?pr=22577#step:5:2434

Aluxio caching test is failing with defaulting parquet writer to UTC in delta
https://github.com/trinodb/trino/pull/22577/commits/306315e5b10b3e60630c665ff64abe925077457b#diff-836fd9091213321cf465d1d29d45862bf927cc60daaa7b4e28713080010b6d4dR500-R501